Update the install guide for Mac (#1312)
pyrsia/pyrsia: docs/tutorials/quick-installation.mdx

```shell
brew install pyrsia
```
- Open a terminal and start Pyrsia Node as background service using the following
```shell
brew services start pyrsia
```

> As per service configuration, Pyrsia Node service logging level set to `info` except module pyrsia set to `dubug`.

Reviewer: typo
```suggestion
As per service configuration, Pyrsia Node service logging level set to `info` except module pyrsia set to `debug`.
```

```

> - Open a terminal and start Pyrsia Node using the following

Reviewer 2: ```suggestion
- Open a terminal and start Pyrsia Node as background service using the following
```

```shell
brew services start pyrsia
```

> This will start Pyrsia node in you local machine with logging level set to `info`. (For additional level of logging, you can also set this to `export RUST_LOG=debug`.)

Reviewer 2: ```suggestion
As per service configuration, Pyrsia Node service logging level set to `info` except module pyrsia set to `dubug`.
```

> When you start the Pyrsia node for the first time, a directory is created in `/usr/local/var/pyrsia/` where all logs and blockchain data are saved. Also macOS might ask for network connection permission. So make sure to
> allow this.

Reviewer 2: ```suggestion
When you start the Pyrsia node for the first time, a directory is created in `/usr/local/var/pyrsia/` where all logs and blockchain data are saved. Also macOS might ask for network connection permission. So make sure to allow this.
```

- Once Pyrsia node is running state, similar kind of logs can be found in the Homebrew log file.
```

>  $ tail -f /usr/local/var/pyrsia/logs/stdout/pyrsia_node.log

Reviewer 2: ```suggestion
 $ tail -f $(brew --prefix)/var/pyrsia/logs/stderr/pyrsia_node_err.log

```

Author: I merged https://github.com/pyrsia/homebrew-pyrsia/pull/4 (Maybe misunderstood sorry) so should we just avoid hard coding?

- Open a terminal and start Pyrsia Node using the following
```shell
export RUST_LOG=info
brew services start pyrsia
```

> This will start Pyrsia node in you local machine with logging level set to `info`. (For additional level of logging, you can also set this to `export RUST_LOG=debug`.)

Reviewer 3: Get rid of RUST_LOG=info both in the shell as in the text. RUST_LOG is set by the service

> When you start the Pyrsia node for first time, a directory named pyrsia is created in the current directory. Also macOS might ask for network connection permission. So make sure to

Reviewer 3: "directory named pyrsia" better specify the full path here now since this isn't in the current dir anymore

allow this.

- Once Pyrsia node is running state, similar kind of logs can be found in the Homebrew log file.
```

>  $ tail -f /usr/local/var/pyrsia/logs/stdout/pyrsia_node.log

Author: Should check if this path is correct (See the question in this PR description).

Reviewer 3: ⬆️ question for @dasmanas?

Author: In conclusion, we should change the path of logs.
https://github.com/pyrsia/pyrsia/pull/1312#issuecomment-1324340042

Reviewer 2: Hardcoding the directory is little tricky and could be misleading from architecture to architecture (like Intel and ARM). The generic command should be
```
tail -f $(brew --prefix)/var/pyrsia/logs/stderr/pyrsia_node_err.log
```
Refer documentation [Homebrew Variables for directory locations](https://docs.brew.sh/Formula-Cookbook#variables-for-directory-locations)
